src/vaadin-context-menu.html

```diff
@@ -412,6 +412,9 @@
 
             this._oldListenOn.style.webkitTouchCallout = '';
             this._oldListenOn.style.webkitUserSelect = '';
+            this._oldListenOn.style.MozUserSelect = '';
```

Comment:
Could you remove `moz-user-select` as according to  MDN, Firefox supports `webkit` since 49

https://developer.mozilla.org/en-US/docs/Web/CSS/user-select

Reply:
Removed.